docs/docs/cmd/pp/solution/solution-publisher-add.md

```diff
@@ -0,0 +1,49 @@
+# pp solution publisher add
+
+Adds a specified publisher in a given environment.
+
+## Usage
+
+```sh
+m365 pp solution publisher add [options]
+```
+
+## Options
+
+`-e, --environment <environment>`
+: The name of the environment.
+
+`-n, --name [name]`
+: The unique name of the publisher.
+
+`--displayName <displayName>`
+: The friendly name of the publisher
+
+`--prefix <prefix>`
+: The prefix of the publisher
+
+`--choiceValuePrefix <choiceValuePrefix>`
+: The choice value prefix of the publisher
+
+`-a, --asAdmin`
```

Comment:
Let's remove the short option here. For reference, please take a look at the issue #4132. 😄